data/yam/agama/auto/sle_nvme.jsonnet

```diff
@@ -0,0 +1,71 @@
+{
+  product: {
+    id: '{{AGAMA_PRODUCT_ID}}',
+  },
+  user: {
+    fullName: 'Bernhard M. Wiedemann',
+    password: '$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/',
+    hashedPassword: true,
+    userName: 'bernhard',
+  },
+  root: {
+    password: '$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/',
+    hashedPassword: true,
+  },
+  storage: {
```

Comment:
storage should be created automatically like in the interactive one, there is no steps to configure anything, so I'm guessing here that without this key we will get the same result, right? if now, we should ask, perhaps is that this feature is not implemented.